src/octonode/user.coffee

```diff
@@ -15,21 +15,29 @@ class User
     Object.keys(data).forEach (e) =>
       @[e] = data[e]
 
-  # Get a user
-  # '/users/pkumar' GET
+  # Get a user (promise)
   info: (cb) ->
-    @client.get "/users/#{@login}", (err, s, b, h)  ->
-      return cb(err) if err
-      if s isnt 200 then cb(new Error('User info error')) else cb null, b, h
```

Comment:
oddly enough, this looks like it is a last line of defense catch. If there's a 4xx or 5xx, then @client.get returns err. The line:
`if s isnt 200 then`...
only triggers if there wasn't a 400 or 500 _error, _and_ the response isn't giving a statuscode of 200